setup_dependencies.py

```diff
@@ -188,7 +188,11 @@
                      'WMCore.Services+', 'WMCore.WMSpec+',
                      'WMCore.WMBS+', 'WMCore.ResourceControl+'],
         'systems': ['wmc-web', 'wmc-database', 'global-workqueue', 'wmc-runtime'],
-        'statics': ['src/javascript/WMCore/WebTools/Agent',
+        'statics': ['src/couchapps+',
+                    'bin+',
+                    'deploy+',
+                    'etc+',
+                    'src/javascript/WMCore/WebTools/Agent',
```

Comment:
Are you sure that we need this `WebTools`? I am inclined to say that these are legacy and should actually be removed.

Reply:
Then unless you are certain we need it, please remove the one that you added with this PR, which is:
```
'src/javascript/WMCore/WebTools/Agent',
```

Reply:
I did not add it with this PR. It was the original line changed by the very first commit, but anyway I am removing it.

Reply:
Sorry Todor, I missed this detail. If it's not a new line, let it be then. PR is good to go!